compiler/damlc/daml-compiler/src/DA/Daml/Compiler/DataDependencies.hs

```diff
@@ -318,6 +343,15 @@ mkUserTyVar =
 mkRdrName :: T.Text -> Located RdrName
 mkRdrName = noLoc . mkRdrUnqual . mkOccName varName . T.unpack
 
+mkError :: T.Text -> LHsExpr GhcPs
+mkError msg = noLoc $ HsApp noExt
+    (noLoc $ HsVar noExt (mkRdrName "error"))
+    (mkStringLit msg)
+
+mkStringLit :: T.Text -> LHsExpr GhcPs
```

Comment:
Judging from CI this seems to break probably due to `OverloadedStrings`, maybe try generating `{-# LANGUAGE NoOverloadedStrings #-}` or whatever the extension is called to disable this.

Reply:
Trying it out. I spent way too much time looking for where to add this extension in `HsModule` ... turns out there was a plaintext header being added with the list of extensions. 😅